etc/picongpu/hemera-hzdr/defq.tpl

```diff
@@ -97,5 +105,5 @@ export OMPI_MCA_io=^ompio
 
 if [ $? -eq 0 ] ; then
   # Run PIConGPU
-  mpiexec --bind-to none !TBG_dstPath/tbg/cpuNumaStarter.sh !TBG_dstPath/input/bin/picongpu !TBG_author !TBG_programParams
+  source !TBG_dstPath/tbg/handleSlurmSignals.sh mpiexec --bind-to none !TBG_dstPath/tbg/cpuNumaStarter.sh !TBG_dstPath/input/bin/picongpu !TBG_author !TBG_programParams
```

Comment:
Is this a single line on purpose?

Reply:
I can break with \ at the line end. Will add it tomorrow

Reply:
Maybe it's not required. I was mostly wondering if that was an accidental missing end-of-line. It this is intended, could stay as is

Reply:
Wait let me test if I can call it `$(!TBG_dstPath/tbg/handleSlurmSignals.sh mpiexec ...)`
It is required to call it in the same context as the batch script. I tested something else before and accidentally thought I need to use `source`.

Reply:
updated